simcore/simulation/library/rigid_filament.cpp

```diff
@@ -1,4 +1,5 @@
 #include "rigid_filament.hpp"
+#include "function_headers.hpp"
```

Comment:
You can remove this line: function_headers.hpp is included in auxiliary.hpp, which is included by every object by default.

Reply:
Changed